test/test_loader.py

```diff
@@ -180,9 +180,44 @@ def test_load_all_other_type(self):
             mock_log.assert_has_calls(
                         [call('Message contains %i records', 1)])
 
+    def test_load_all_msgs(self):
+        """Check that load_records is called with an empty list."""
+        pidfile = os.path.join(self.dir_path, 'pidfile')
+
+        valid_body = """APEL-summary-job-message: v0.2
+                    Site: RAL-LCG2
+                    Month: 3
+                    Year: 2010
+                    GlobalUserName: /C=whatever/D=someDN
+                    VO: atlas
+                    VOGroup: /atlas
+                    VORole: Role=production
+                    WallDuration: 234256
+                    CpuDuration: 2345
+                    NumberOfJobs: 100
+                    %%"""
+        
+        in_q = dirq.queue.Queue(os.path.join(self.dir_path, 'incoming'),
+                                schema=schema)
+        re_q = dirq.queue.Queue(os.path.join(self.dir_path, 'reject'),
+                                schema=schema)
+
+        in_q.add({"body": "    ", "signer": "   ", "empaid": "3",
+                  "error": ""})
+        in_q.add({"body": valid_body, "signer": "Test signer", "empaid": "4",
+                  "error": ""})
+        in_q.add({"body": "Still it is a Bad Message", "signer": "Test signer", "empaid": "5",
+                  "error": ""})
+
+        self.loader = apel.db.loader.Loader(self.dir_path, True, 'mysql',
+                                            'host', 1234, 'db', 'user', 'pwd',
+                                            pidfile)
+        self.loader.load_all_msgs()
+        self.assertEquals(re_q.count(), 2)
+
     def tearDown(self):
         shutil.rmtree(self.dir_path)
         mock.patch.stopall()
 
 if __name__ == '__main__':
-    unittest.main()
+    unittest.main()
```

Comment:
Can you make sure files follow the Unix convention of having a blank line at the end?

Reply:
I haven't made this particular change as part of the commit. But, I am happy to fix it.